scripts/generateAssetData/avalanche/index.ts

```diff
@@ -17,3 +48,60 @@ export const getAssets = async (): Promise<Asset[]> => {
       }),
   }))
 }
+
+const fetchPortalsTokens = async (): Promise<TokenInfo[]> => {
+  const url = 'https://api.portals.fi/v2/tokens'
+  const PORTALS_API_KEY = process.env.REACT_APP_PORTALS_API_KEY
+  if (!PORTALS_API_KEY) throw new Error('REACT_APP_PORTALS_API_KEY not set')
+
+  const params = {
+    // Maximum supported limit, enough to get a viable diff, but low enough to not make this huge as a PoC
+    limit: '250',
+    // Only Avalanche for PoC, more to be added later
+    networks: ['avalanche'],
+  }
+
+  try {
+    const response = await axios.get<GetTokensResponse>(url, {
+      // Encode query params with arrayFormat: 'repeat' because Portals api expects it
+      paramsSerializer: params => qs.stringify(params, { arrayFormat: 'repeat' }),
+      headers: {
+        Authorization: `Bearer ${PORTALS_API_KEY}`,
+      },
+      params,
+    })
+    const tokens = response.data.tokens
```

Comment:
Ensure we filter this guy out 
<img width="673" alt="image" src="https://github.com/shapeshift/web/assets/17035424/f0951b0e-946a-41df-b0d4-f60e90f57e9f">

Reply:
9aec9a6f00